e2e/cypress/support/api_commands.js

```diff
@@ -353,6 +353,49 @@ Cypress.Commands.add('createNewUser', (user = {}, teamIds = []) => {
     });
 });
 
+/**
+ * Creates a new user via the API, adds them to a team, but does not set preference to bypass tutorial.
+ * Then logs in as the user
+ @param {Object} user - Object of user email, username, and password that you can optionally set. Otherwise use default values
+ @returns {Object} Returns object containing email, username, id and password if you need it further in the test
+ */
+Cypress.Commands.add('createNewUserWithTutorial', (user = {}, teamIds = []) => {
```

Comment:
I see that following methods have been added additionally in the api_commands.js, i.e. `createNewUserWithTutorial` and `loginAsNewUserWithTutorial`. Rather than having seperate functions for these, we can pass an additional parameter skipTutorial to the existing functions `createNewUser` and `loginAsNewUser` and by default set it to True, so that by default we skip the tutorial. You can then conditionally check if this flag is true and only then execute the steps to skip the tutorial in the `createNewUser` function. This way, if there are changes in the login process, we do not have to update multiple functions.

Reply:
I removed these methods and added a Boolean flag, bypassTutorial, with a default of True to the existing methods.  I added the flag to this test and retested existing tests, and they seem to work as expected.